src/operators/reduce_window.rs

```diff
@@ -20,7 +20,9 @@ pub(crate) struct ReduceWindowLogic {
 impl ReduceWindowLogic {
     pub(crate) fn builder(reducer: TdPyCallable) -> impl Fn(Option<StateBytes>) -> Self {
         move |resume_acc_bytes| {
-            let acc = resume_acc_bytes.map(|resume_acc_bytes| resume_acc_bytes.de());
+            let acc = resume_acc_bytes
+                .map(StateBytes::de::<Option<TdPyAny>>)
+                .flatten();
```

Comment:
It was actually needed, hence the panic, but type inference made the compiler assume the wrong types since it can't know what's in those bytes.

The previous version without the flatten was causing the compiler to infer the incorrect type for the deserialization step. So for reduce window, `snapshot` was serializing with `StateBytes::ser(&self.acc)` which was inferred (correctly) to be `StateBytes::ser::<Option<TdPyAny>>(&self.acc)` _but_ the builder was deserializing with `resume_acc_bytes.map(|resume_acc_bytes| resume_acc_bytes.de())` which was inferred to be `resume_acc_bytes.map(|resume_acc_bytes| resume_acc_bytes.de::<TdPyAny>())` but by (bad) luck the final type of `acc` was correct, so it was like cool this must be correct.

By making the deserialization type explicit and match the serialization type, now if you forget the `flatten` it's a type error.